brainzutils/test/test_mail.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+import smtplib
+
+from mock import patch, call
+from brainzutils import flask
+from brainzutils import mail
+
+
+class SendEmailTests(unittest.TestCase):
+
+    @patch('brainzutils.mail.send_mail')
```

Comment:
If you mock the brainzutils send mail function, then there is no point of this test. You should try to mock out the function that generates the smtp object, hence getting a MagicMock object in place of the smtp object and then check that the smtp send_mail function is called correctly.